site/content/blog/2020-11-01-whats-new-in-svelte-november-2020.md

```diff
@@ -0,0 +1,43 @@
+---
+title: What's new in Svelte: November 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+BLURB SUMMARIZING WHAT'S NEW...
+"What's new in Svelte" series! We'll try to make this a monthly blog post in which you'll find out about new features, bug fixes, and a showcase of Svelte projects from around the community.
+
+## New features
+1. 
+
+**What's going on in Sapper?**
+Any updates here?
```

Comment:
0.28.10 was released: https://github.com/sveltejs/sapper/blob/master/CHANGELOG.md#02810

Reply:
Awesome! I'll read through and pull out some highlights! If you have any suggestions / high level summary, I'd be happy to include any blurb you'd like :)